front/src/modules/ui/components/buttons/Button.tsx

```diff
@@ -47,7 +51,18 @@ const StyledButton = styled.button<
         return 'none';
     }
   }};
-  border-radius: 4px;
+  border-radius: ${({ position }) => {
+    switch (position) {
+      case 'left':
+        return '4px 0px 0px 4px';
```

Comment:
@FelixMalfait Instead of using numbers like `4`, `theme.spacing(1)` helper should be used